spinalcordtoolbox/reports/qc.py

```diff
@@ -836,7 +848,14 @@ def qcslice_layout(x): return x.single()
         def qcslice_layout(x): return x.single()
     # Metric outputs (only graphs)
     elif process in ['sct_process_segmentation']:
-        assert os.path.isfile(path_img)
+        plane = 'Sagittal'
+        dpi = 100  # bigger picture is needed for this special case, hence reduce dpi
+        fname_list = [fname_in1]
+        fname_list.extend(fname_seg)
```

Comment:
I think this line is the cause of the extra image in `mask` that is never used...